JS/Lesson15/gulcin/script.js

```diff
@@ -113,52 +117,63 @@ console.log('------------------------');
 // 16. Extract the price and currency from a string below using "slice" method and print it to console.
 const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
 
-console.log('Exercise16: ' /* Result here */);
+console.log('Exercise16: ' , planPriceString.slice(planPriceString.indexOf('9'), -7));
 console.log('------------------------');
 
 // 17. Use the `repeat` method to create a string of asterisks (*) that forms a horizontal line with a length of 10 and print it to the console.
 
-console.log('Exercise17: ', /* Result here */);
+console.log('Exercise17: ', '*'.repeat(10));
 
 console.log('------------------------');
 // 18. Use the `concat` method to combine three separate strings into one and print the result to the console.
 const string1Exercise18 = 'Hello, ';
 const string2Exercise18 = 'world';
 const string3Exercise18 = '!';
-console.log('Exercise18: ', /* Result here */);
+console.log('Exercise18: ', string1Exercise18.concat(string2Exercise18, string3Exercise18));
 
 console.log('------------------------');
 // 19. Use the `split` and `join` methods to reverse the words in the string stringExercise28 and print the result to the console.
 const stringExercise19 = 'This is a sample sentence.';
-console.log('Exercise19: ', /* Result here */);
+console.log('Exercise19: ', stringExercise19.split('').reverse().join(''));
```

Comment:
There's a minor nuance in here. You need to reverse words in the sentence, not the entire sentence. 
The error is here: `stringExercise19.split('')` should be `stringExercise19.split(' ')` (break the sentence on the spaces to separate the words).

Then, when doing `join`, don't forget to add spaces again!

Reply:
I thought we were going to reverse the words also, I changed the correct answer :))